inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -412,7 +412,8 @@ void GNAPlugin::InitGNADevice() {
                 config.swExactMode,
                 gnaFlags->gna_lib_async_threads_num,
                 gnaFlags->gna_openmp_multithreading,
-                gnaFlags->performance_counting);
+                gnaFlags->performance_counting,
+                !config.dumpXNNPath.empty());
```

Comment:
consider to additionally pass `GnaPlugin::GetDeviceVersionFromString(config.dumpXNNGeneration);`